99-Swiss officer ratios.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plot_swiss_officer_ratios_by_group(new_officers_graph, lambda G: k_clique_communities(G, n), f'{n}-clique')
</pre><p>I dont know what this function does but I think its not equivalent to n-cliques in the san script.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>